### DISCUSSION TITLE
Approved, but benefits is not available.

### DISCUSSION THREAD

**Discussion by rickybharti** [community/community#137923] [GitHub Education]
### Hi everyone,

It's been already a few days, please someone approve it.

**Comment by donbt2** [comment]
Me too, but I didn't want to create a new thread

**Comment by davevad93** [comment]
Hi @rickybharti ,

What's the status of your application?

If the label is green and says "`Congrats! Your request for @rickybharti was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and says "`Congrats! Your request for @rickybharti was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack) and select one of the available offers, if you can access them you are fine, if not I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

**Reply by Manpreet0980** [reply]
few days ago and my application is approved but my perk not available 
Plz sort out

**Comment by myh66** [comment]
+1

**Comment by Gamewallah63** [comment]
I am a verified student having trouble redeeming my Global Campus offers . Approved few days but still I don't get Benefit till now
 Please make it fast ⏩
